cloud-core/src/main/java/cloud/commandframework/exceptions/NoPermissionException.java

```diff
@@ -67,11 +67,25 @@ public final String getMessage() {
      * Get the missing permission node
      *
      * @return Get the missing permission node
+     * @deprecated use {@link #missingPermission()} instead
      */
+    @API(status = API.Status.DEPRECATED, since = "1.8.5")
+    @Deprecated
     public @NonNull String getMissingPermission() {
         return this.missingPermission.toString();
     }
 
+    /**
+     * Returns the missing {@link CommandPermission}
+     *
+     * @return the missing permission
+     * @since 1.8.5
+     */
+    @API(status = API.Status.STABLE, since = "1.8.5")
```

Comment:
new API should target 1.9

Reply:
okay cool - `1.9.0` ?

Reply:
and the version for the deprecation annotation should also be `1.9.0` ?

Reply:
yes and yes